client/src/main/java/org/opensearch/ml/client/MachineLearningClient.java

```diff
@@ -322,8 +322,21 @@ default ActionFuture<DeleteResponse> deleteConnector(String connectorId) {
         return actionFuture;
     }
 
+    /**
+     * Delete connector for remote model
+     * @param connectorId The id of the connector to delete
+     * @return the result future
+     */
+    default ActionFuture<DeleteResponse> deleteConnector(String connectorId, String tenantId) {
```

Comment:
Do we need similar ML client for create/Get/update using tenant ID?

Reply:
I think so, this is when flow framework will make the transport call to ml-commons. I kept the old one for backward compatibility. Do you have suggestion to do this differently?